acra-core/src/main/java/org/acra/startup/StartupProcessorExecutor.kt

```diff
@@ -48,7 +48,11 @@ class StartupProcessorExecutor(private val context: Context, private val config:
                         when {
                             report.delete -> if (!report.file.delete()) warn { "Could not delete report ${report.file}" }
                             report.approved -> send = true
-                            report.approve && isAcraEnabled -> ReportInteractionExecutor(context, config).performInteractions(report.file)
+                            report.approve && isAcraEnabled -> {
+                                    ReportInteractionExecutor(context, config).performInteractions(report.file)
```

Comment:
performInteractions returns a boolean indicating if reports can be sent immediately or not, this needs to be respected here as well.

Reply:
I see. What if I added this verification, similar to what is seen in ReportExecutor:

```
if (ReportInteractionExecutor(context, config).performInteractions(report.file)){
     schedulerStarter.scheduleReports(report.file, false)
}
```

Reply:
Let me know if anything else should be addressed